docs/en/ingest-management/elastic-agent/otel-agent-configuration.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[otel-agent-configuration]]
+= {agent} OTel configuration
+
+preview::[]
+
+When running in `otel` mode, {agent} supports a standard OTel Collector configuration. 
```

Comment:
Would it help to provide a link to the upstream docs on configuration? https://opentelemetry.io/docs/collector/configuration/

Reply:
Oh now I see it is linked down below. :slightly_smiling_face: